### PR TITLE
[Translation] [Indonesian]: /id/guide/games.md

### PR DESCRIPTION
**Pull request by CikiMomogi**
Ready for review.

### CODE REVIEW

File: site/docs/id/guide/games.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+     // Kamu bisa mendapatkan id chat pengguna menggunakan `ctx.from.id`
+     // Ia akan menyediakan id chat pengguna yang memulai command start tersebut
+     const chatId = ctx.from.id;
+     await ctx.api.sendGame(chatid, "gathotkaca_reborn");
+   });
+   ```
+
+> [Ingat!](./basics.md#mengirim-pesan) Saat mengirim pesan, kamu bisa menentukan opsi lebih lanjut menggunakan opsi-opsi yang disediakan oleh object type `Other`.
+
+Kamu juga bisa mengatur [inline keyboard](../plugins/keyboard.md#inline-keyboard) untuk menampilkan tombol khusus di pesan game tersebut.
```
Comment (CikiMomogi) [inline site/docs/id/guide/games.md:54]: - [ ] Fix this anchor link

File: site/docs/id/guide/games.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+```ts
+// Masukkan url game yang sudah di-hosting di web di sini.
+bot.on("callback_query:game_short_name", async (ctx) => {
+  await ctx.answerCallbackQuery({ url: "url_game_kamu" });
+});
+```
+
+---
+
+### Hasil Akhir Kode Kita Akan Terlihat Seperti Ini
```
Comment (fadzikri) [inline site/docs/id/guide/games.md:88]:
```suggestion
### Hasil Akhir Kode Kita akan Terlihat seperti Ini
```

File: site/docs/id/guide/games.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+  await ctx.replyWithGame("game_pertamaku", {
+    reply_markup: keyboard,
+    // kamu bisa menggunakan method api di sini, sesuai dengan kebutuhanmu.
+  });
+});
+```
+
+> Jangan lupa untuk memasang [error handler](./errors.md) ke bot kamu sebelum diluncurkan ke publik.
+
+Di masa mendatang, kami mungkin akan melanjutkan artikel ini di materi tingkat lanjut dan FAQ, meski begitu kami rasa materi ini sudah cukup untuk kamu bisa menjalankan game di Telegram.
```
Comment (fadzikri) [inline site/docs/id/guide/games.md:105]:
```suggestion
Di masa mendatang, kami mungkin akan melanjutkan artikel ini di materi tingkat lanjut dan FAQ, meski begitu kami rasa materi ini sudah cukup untuk dapat menjalankan game-mu di Telegram.
```

**Review by fadzikri — CHANGES_REQUESTED** [review]
Improvitation?

File: site/docs/id/guide/games.md (hunk 2 above)
Comment (CikiMomogi) [reply site/docs/id/guide/games.md]: The word `Akan` based on [KBBI](https://kbbi.kemdikbud.go.id/entri/akan) is not a particle, so it should be capitalized.

**Pull request merged**